packages/react/src/lib/accessibility/FocusZone/FocusZone.types.ts

```diff
@@ -32,111 +32,137 @@ export interface IFocusZone {
  * FocusZone component props.
  */
 export interface FocusZoneProps extends React.HTMLAttributes<HTMLElement | FocusZone> {
+  /**
+   * Element type the root element will use. Default is "div".
+   */
+  as?: React.ReactType
   /**
    * Additional class name to provide on the root element, in addition to the ms-FocusZone class.
    */
   className?: string
 
   /**
    * Defines which arrows to react to.
+   * It has next options: horizontal, vertical, bidirectional.
    * @default FocusZoneDirection.bidirectional
    */
   direction?: FocusZoneDirection
 
   /**
-   * Function which uses root element to select and return the intial tabbable element.
+   * Function which uses root element as parameter to return the intial tabbable element.
+   * For example, when there is a chat with a bottom-up approach, it is expected that the last chat message is tabbable (active), not the first default one.
+   * @default undefined
    */
   defaultTabbableElement?: (root: HTMLElement) => HTMLElement
 
   /**
    * If a default tabbable element should be force focused on FocusZone mount.
+   * @default false
```

Comment:
I suggest to patch our plugin instead, something like `if (type === 'boolean' && default === undefined) default = false`

Reply:
sorry, didn't get 🙃